.gitignore

```diff
@@ -96,3 +96,5 @@ _output
 
 # controller-runtime testenv binaries
 testbin/*
+
+3scale-operator
```

Comment:
hey Eguzki, why are we adding 3scale-operator to gitignore?

Reply:
My IDE (vim) when compiling the code from the main.go generates the binary `3scale-operator` file at the root of the project.

Reply:
cool, thanks